### ISSUE TITLE
Allow usage of libQt5Bluetooth.so.5

### ISSUE THREAD

**Issue by martonmiklos** [sailfishos/sdk-harbour-rpmvalidator#99]
Title says all: no apps could be published to the Harbour which would utilise the Bluetooth. 

With the boom of the BLE devices and the Bluez 5 supported devices it would be nice to allow these apps into the Harbour.

**Comment by jdrescher2006** [comment]
Come on guys it's 2017 and there is no bluetooth allowed? 
Can someone even explain why this is the case?
I have two apps waiting to go to jolla store, only thing needed is bluetooth.

**Comment by martonmiklos** [comment]
@jdrescher2006 
I agree, and I would like to have some comment on it from the Jolla Employees. 
Is there any blocking task with we can help with for e.g.

**Comment by jdrescher2006** [comment]
No answer from a representativ or anyone, this is exactly what I expected.
It's so sad but absolutely typical.
It's the same with e.g. paid apps. Requested since years but not even an inofficial statement why we don't get them. Just no communication at all and that's it.
You know, these are the main blockers keeping SFOS down: important libs (which do exist and work since years!!!) which are not allowed and missing paid apps.
Without these, there will never ever be high class apps! Why should a dev even bother?

**Comment by martonmiklos** [comment]
@jdrescher2006 
Well I would not position this problem to the same category as the paid apps. For the later there are a huge bunch of work which needs to be done in several places (Harbour app, payment methods, etc.). 

But this is just the matter of allowing the usage of an existing library. 
And yeah a comment like:
* we do not want to do it (everyone should roll with his own implementation)
* we do not have bandwidth for verification right now
* whatever else
would be sufficient for me at the moment.

**Comment by jpetrell** [comment]
We haven't allowed QtBluetooth before since it is not a very mature Qt module and suffers from bunch of issues that we would like to fix in Qt first, but mainly because we really haven't had bandwidth. We are now actively discussing it though, so there is always hope. :)

**Comment by martonmiklos** [comment]
Thanks for the explanation!

**Comment by jpetrell** [comment]
Help verify QtBluetooth API works on top of latest Sailfish OS release (e.g. Jolla C + Bluez 5):
- QBluetoothLocalDevice test information queries, powering on/off, pairing/unpairing
- QBluetoothDeviceDiscoveryAgent discover nearby devices, check that correct device info is exposed
- QBluetoothServer can be used to communicate with Bluetooth device
- QBluetoothServiceDiscoveryAgent returns valid services and service info
- QBluetoothTransferManager can be used to transfer data to another device

Refactor Qt Bluetooth to get rid of synchronous D-Bus calls (counted 58 QDBusPendingReply::waitForFinished calls in qtconnectivity/src/bluetooth)

**Comment by martonmiklos** [comment]
Thanks Joona!
So you are more curious about the non Low Energy stuff?
For the LE part I have compiled the Qt example, and it works pretty well according to my tests with various devices:
https://openrepos.net/content/martonmiklos/ble-scanner

**Comment by jdrescher2006** [comment]
Just a little reminder. It's well into 2018 now and still no bluetooth in Jolla store. No classic and no BLE.
By the way, for me it' a pure myth to say QtBluetooth is not mature. I have 2 apps with bluetooth classic support in openrepos. They have hundreds of users on different phones. There are NO bluetooth problems whatsoever -> pure myth and distraction on behalf of Jolla.
Face the thruth: you guys got your priorities completely messed up! How you were able to achieve this OS with all these basics messed up is an eternal mysterium for me.
I will write again if it's 2019. See you later...

**Comment by Thaodan** [comment]
Update for 2019?
Soon this bug will be 3.
